docs/content/policy-reference.md

```diff
@@ -975,6 +975,7 @@ The table below shows examples of calling `http.send`:
 
 | Built-in | Description | Wasm Support |
 | ------- |-------------|---------------|
+| <span class="opa-keep-it-together">``net.lookup_ip_addr(name)``</span> | `output` is a set of IP addresses (both v4 and v6, strings) that the domain name resolves to using standard name resolution. | ``SDK-dependent`` |
```

Comment:
The context you provided about caching and netgo vs cgo in the PR message might be useful here. WDYT?